apps/debate/views.py

```diff
@@ -96,7 +96,7 @@ def get_permission_object(self):
 class SubjectDeleteView(
     mixins.DashboardBaseMixin,
     mixins.DashboardComponentMixin,
-    mixins.DashboardComponentDeleteSignalMixin,
+    mixins.DashboardComponentFormSignalMixin,
```

Comment:
@goapunk this has been deprecated for a while. and we had left a trial period when we switched to `form_valid` for deletion.